ci/workflows/main.yml

```diff
@@ -0,0 +1,103 @@
+steps:
+  - label: "compute node build dev"
+    command: "ci/scripts/compute-node-build.sh -t debug -p dev"
+    key: "dev"
+    plugins:
+      - docker-compose#v3.9.0:
+          run: rw-build-env
+          config: ci/docker-compose.yml
+      - artifacts#v1.5.0:
+          upload:
+            - from: target/debug/risingwave
+              to: risingwave-dev
+            - from: target/debug/risingwave_regress_test
+              to: risingwave_regress_test-dev
+            - from: target/debug/risedev-playground
+              to: risedev-playground-dev
+    timeout_in_minutes: 10
+
+  - label: "compute node build release"
+    command: "ci/scripts/compute-node-build.sh -t release -p release"
+    if: build.source != "schedule"
+    key: "release"
+    plugins:
+      - docker-compose#v3.9.0:
+          run: rw-build-env
+          config: ci/docker-compose.yml
+      - artifacts#v1.5.0:
+          upload:
+            - from: target/release/risingwave
```

Comment:
can we have something like a job matrix to run {debug|release} jobs without writing them twice?

Reply:
Buildkite matrix values can only inside command step attributes, labels and commands. Interpolation of matrix values inside plugin attributes is not supported.